RawWaterManagement/OpenChannelJunction/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,113 @@
+{
+  "location": {
+    "type": "Property",
+    "value": {
+      "type": "Point",
+      "coordinates": [
+        -60.6983145,
+        9.005378
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Protect change approach key glass suddenly. President share to for traditional shake. Raise whole raise perhaps focus within first.",
+      "addressLocality": "Page there your seven. Forward fire recent country team. Nearly page Republican consumer similar. Son author itself day business cold.",
+      "addressRegion": "Participant exactly ten policy join. Former artist during pattern. Represent form cover stuff star record receive.",
+      "addressCountry": "Fast into brother see his keep discover. Different individual film sense.",
+      "postalCode": "Place go occur people south. Third site material particularly. Evidence industry red tend focus fly mind these.",
+      "postOfficeBoxNumber": "She everybody actually more true. East play decide before style. He fear body thank effect above.",
+      "areaServed": "Dream newspaper foreign coach attorney true economy. Skin film natural mission."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Floor nearly stand always right. Ready information skin oil boy."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:PYGQ:66276695",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1978-08-03T08:57:03Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2021-03-23T00:16:00Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Rate concern party level food."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Situation fast ok including report answer capital note. Decide eat learn. Price kitchen why would."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Home customer data can after. Degree too value into some wall whom."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Able dog national whose measure consider source. Quality democratic peace actually remember study focus. Myself court bring young others assume business."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "They agency impact my. Apply movement fast ten without serious long."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:WNCG:70269691",
+      "urn:ngsi-ld:dataModel:items:INOD:84827103"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:NHOU:08830312",
+      "urn:ngsi-ld:dataModel:items:ULRL:23255556"
+    ]
+  },
+  "type": "OpenChannelJunction",
+  "position": {
+    "type": "Property",
+    "value": {
+      "distance": 864.6,
+      "refPoint": "urn:ngsi-ld:dataModel:refPoint:JXFD:60487647"
+    }
+  },
+  "downstreamNode": {
```

Comment:
type should be Relationship
The proper way to model Relationships is as follow :

"observedBy":{
	"type":"Relationship",
	"object":" urn:ngsi-ld:Device:01 "	}